### PR TITLE
[webui][api] update release notes for public key/ssl cert download

### PR DESCRIPTION
**Pull request by evanrolfe**
update release notes to mention this new feature:
https://github.com/openSUSE/open-build-service/pull/2544

### CODE REVIEW

**Comment by adrianschroeter** [comment]
is "ssl cert" correct english? *g*

Or should we use "SSL certificate" term here? ;)

**Comment by evanrolfe** [comment]
@adrianschroeter I've changed it to "certificate" now

File: ReleaseNotes-2.8 [hunk 1]
```diff
@@ -39,6 +39,8 @@ Features
  * /about route is now independent from request format
  * workers are now also tracked when they went away (new states "down", "away" and "dead")
  * Main projects list is now filtered based on a configurable (by the admin) regular expression
+ * Users can download the public key and SSL certificate for a project via the project home page
+ * New API route to get public key and SSL certificate: GET /source/:project_name/:_keyinfo
```
Comment (mdeniz) [inline ReleaseNotes-2.8]: The route is correct?, I mean the `_keyinfo` is without `:` ? GET /source/:project_name/_keyinfo

Comment (evanrolfe) [inline ReleaseNotes-2.8]: ah good spot, fixing..

**Comment by mschnitzer** [comment]
lgtm 👍

**Pull request merged**